src/components/core/feeds/ThreadPreview.tsx

```diff
@@ -137,8 +103,15 @@ const ThreadPreview: React.FC<{
     };
   }, [threadId]);
 
+  // If the thread is hidden, use a special placeholder to represent it.
   if (thread.hidden) {
-    return <HiddenThread thread={thread} />;
+    let hiddenThreadArgs : HiddenThreadProps = {
```

Comment:
You should pass these directly to the component:

`<HiddenThread threadId={thread.id} boardId={board.id} hidden={!thread.hidden} ... />`

See the components library comment for renaming.